service_container/factories.rst

```diff
@@ -293,5 +293,138 @@ previous examples takes the ``templating`` service as an argument:
                 ->args([ref('templating')])
             ;
         };
+        
+Usage example
+--------------------
+
+The following example is intended to show how to create and use a factory method in Symfony framework.
+Suppose you want to realize the factory method pattern for services, that describe two delivery methods - DPD and SmartPost.
+
+Services (subclasses) definition::
+
+    // src/Deliveries/DPD.php
+    namespace App\Deliveries;
+
+    class DPD
```

Comment:
Let’s use DHL and UPS, it’s more  international

Reply:
Thanks for advise, changes have been introduced.